migration/migrator/migrations/system/20210722131039_change_autograding_workers_json_permission.py

```diff
@@ -0,0 +1,26 @@
+"""Migration for the Submitty system."""
```

Comment:
Is there every a case where we would run this migration, but not the chmod / chown on this file in `INSTALL_SUBMITTY_HELPER`? If not, then the migration can be removed and just let the `INSTALL_SUBMITTY_HELPER` handle the permissions.

Reply:
I have moved it to CONFIGURE_SUBMITTY.py

Reply:
Barb said she would see where the permission change should happen so we shouldn't merge this yet.